### DISCUSSION TITLE
Highlight a place the link moved the document to

### DISCUSSION THREAD

**Discussion by Diabl0pl** [sumatrapdfreader/sumatrapdf#3446] [Q&A] [answered]
Hello,

Is it possible to somehow mark / highlight the place that the link clicked in the document moved me to? The documents I view show electrical schematics so they have hundreds of positions that are linked between each other, when I clicked one of them I am moved to another page, but that page has so many places to look at that it is difficult to exactly find the spot that was linked to the clicked element. Is it possible to highlight the location in some way?

**Comment by GitHubRulesOK** [comment]
Hyperlinks can be set many ways such as page number (XYZ all null so top left of screen is new datum) or they may roll down to a division line such as top left of paragraph, or possibly to an XY + Zoom but those are much more variable as it does not alter the frame so to zoom to such a location it will be marked as at top left of screen.

The main problem is dependent on current zoom factor so when taken to a target point the view right and bottom is anchored such that top left cannot be the desired target.

Try this sample at any zoom the first bookmark will always be the page level
![image](https://user-images.githubusercontent.com/12754913/232770243-71316857-e639-4413-8c7b-87779f3950bd.png)

[sample-hyperjump.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/11261504/sample-hyperjump.pdf)

Zoom in to 150% and the second link will be the word Page (Without zoom it will be in the page)
![image](https://user-images.githubusercontent.com/12754913/232770652-e47a8c87-a4de-414e-be38-960c238379d5.png)

There is an error in current pre-release click on the TEXT word "Page" goes to the wrong page height position it scrolls to X but at bottom not top .

**Comment by Diabl0pl** [comment]
Thanks for the reply. Do you know if it is possible to mark the location with a different background colour or to draw a rectangle like in Adobe Reader? I think this functionality would be very beneficial.

**Reply by GitHubRulesOK** [reply]
Is that a new feature in Acrobat? Its not there in the Edge Reader nor in my older Acrobat
![image](https://user-images.githubusercontent.com/12754913/232830587-41d342b0-fe6a-4390-a2e5-11ad0182906c.png)

![image](https://user-images.githubusercontent.com/12754913/232826126-ca0f943c-da10-432c-aad1-0056c0f90db9.png)

**Comment by Diabl0pl** [comment]
Hey!
Here's a video showing I'm talking about exactly. :)

https://user-images.githubusercontent.com/8557443/232989372-1e09ae20-7c37-417a-9f56-c81894612aef.mp4

**Reply by GitHubRulesOK** [reply]
does that work in the MS Edge powered by Acrobat viewer, e.g. is it a feature of file behaviour or is it a script in Adobe Acrobat that can not be used in other viewers?

Is there a setting in newer acrobat that shows destination for all files from other sources e.g. show how it works with the sample above that has no scripting exploits. It is a simple "goto"

**Comment by Diabl0pl** [comment]
Apparently, it doesn't work this way in MS Edge, it just jumps between places. Also, Adobe Acrobat does not highlight the place in the sample file that you provided, I thought that this is a built in feature of the software. The files that I'm using is generated by "EPLAN", so I guess they are pretty common in the industry.

**Reply by GitHubRulesOK** [reply, marked as answer]
it is possible for an application to write a PDF scripted behaviour such as say flash on and off so I am "guessing"  (without a sample to destroy :-) that is a scripted behaviour and for several reasons (including security) scripts are never run in SumatraPDF.

**Reply by Diabl0pl** [reply]
Ok, thank you very much for the explanation!
